src/main/java/in/projecteka/consentmanager/consent/ConsentManager.java

```diff
@@ -121,24 +127,46 @@ private String getHIUId(ConsentDetail consentDetail) {
         return signedObject.getSignature();
     }
 
-    private ConsentArtefact mapToConsentArtefact(ConsentRequestDetail consentRequest, GrantedConsent consent) {
+    private ConsentArtefact mapToConsentArtefact(ConsentRequestDetail requestDetail, GrantedConsent granted) {
+        PatientLinkedContext linkedPatientInfo = PatientLinkedContext.builder()
```

Comment:
```suggestion
        PatientLinkedContext patientContexts = PatientLinkedContext.builder()
```
Having Info, Detail, etc are kind of extra noise in variables and class type I feel.

For example, if the class name Patient,  object of type Patient should describe the patient, instead of having additional Info suffix with class or object name.

Reply:
Done. No more PatientLinkedContext